src/quo2/components/dropdowns/network_dropdown/view.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.dropdowns.network-dropdown.view
+  (:require [quo2.theme :as quo.theme]
+            [quo2.components.dropdowns.network-dropdown.style :as style]
+            [react-native.core :as rn]
+            [quo2.foundations.resources :as quo.resources]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [reagent.core :as reagent]))
+
+(def networks-list
```

Comment:
Can be `(def ^:private networks-list`
Also I think it might be better to have this in a more shareable place because I think it might be used by multiple components.

Reply:
cc @OmarBasem, what did we do for this in other components? :)

Reply:
hmm, did you check with the design team that these networks are hard coded into it and not changing? Is this not something we will provide at the app level? 🤔 it could change for test networks etc ?

Reply:
@J-Son89, yes, you are probably right, asked designers)

Reply:
@J-Son89 what do you mean hard coded? I think this is importing the network icons

Reply:
@J-Son89 I am not sure if this component uses only 3 networks, but we do have colors for 7 networks. Better be checked with the design team

Reply:
@J-Son89, @OmarBasem, designers have confirmed that list of networks should be modifiable, so I've updated the component to receive networks as an additional parameter